config/nginx/dice-staging.marti.triplea-game.org

```diff
@@ -0,0 +1,41 @@
+server {
+  listen 80;
+  listen [::]:80;
+  server_name dice-staging.marti.triplea-game.org;
```

Comment:
Just curious why we need the fourth level of domain here.  Would `marti.triplea-game.org` and `marti-staging.triplea-game.org` not work?

Reply:
This would work as well, but I wanted to keep the start of the subdomain the same. Just a personal preference.

Reply:
Okay, I'll buy that.  But then why the extra `marti` subdomain?  That is, why not just `dice.triplea-game.org` and `dice-staging.triplea-game.org`?  Having four subdomains seems to be inconsistent when compared to the other services, e.g. `forums.triplea-game.org`, `lobby.triplea-game.org`, etc.

Reply:
Because my intention was to use the shorter variant for the future REST-based dice server 😅